src/register/RegistrationPage.jsx

```diff
@@ -176,12 +218,15 @@ const RegistrationPage = (props) => {
   useEffect(() => {
     if (registrationResult.success) {
       // This event is used by GTM
-      sendTrackEvent('edx.bi.user.account.registered.client', {});
+      sendTrackEvent('edx.bi.user.account.registered.client', {
+        variation: simplifyRegistrationExpVariation,
+        picked_suggested_username: usernameSuggestionsBackup.includes(registrationResult?.authenticatedUser?.username),
```

Comment:
I am not sure this ```registrationResult?.authenticatedUser?.username``` would return the username i guess we need to change it, correct me if I am wrong please

Reply:
I think the the authenticatedUser will have the username in the response. https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/user_authn/views/register.py#L593